Model/Site.hs

```diff
@@ -38,3 +39,6 @@ destroySite siteId = do
     deleteWhere [CommentSite ==. siteId]
     deleteWhere [MembershipSite ==. siteId]
     delete siteId
+
+overSiteQuota :: Int -> Plan -> Bool
```

Comment:
Looks like this is always called with `length sites` - thoughts on changing the type to `[a] -> Plan Bool`?